SpatialGDK/Source/SpatialGDK/Public/SpatialView/OpList/EntityComponentOpList.h

```diff
@@ -31,6 +32,20 @@ class EntityComponentOpListBuilder
 	EntityComponentOpListBuilder& RemoveComponent(Worker_EntityId EntityId, Worker_ComponentId ComponentId);
 	EntityComponentOpListBuilder& SetAuthority(Worker_EntityId EntityId, Worker_ComponentId ComponentId, Worker_Authority Authority);
 	EntityComponentOpListBuilder& SetDisconnect(Worker_ConnectionStatusCode StatusCode, const FString& DisconnectReason);
+	EntityComponentOpListBuilder& AddCreateEntityCommandResponse(Worker_EntityId EntityID, Worker_RequestId RequestId,
+																 Worker_StatusCode StatusCode, const FString& Message);
```

Comment:
Wonder if it might be better to make these StringStore arugments rather than FString. As we never actually want anything here to be utf16 encoded and it's implicitly convertible by design.